### DISCUSSION TITLE
Can't use touch gestures even when using custom builds

### DISCUSSION THREAD

**Discussion by HariShyamU** [fabricjs/fabric.js#7518] [Q&A]
I've been trying to get a FabricJs canvas work with multitouch pan and zoom, but to no avail. I've tried countless custom builds but the event doesn't have any touch information to work on. Here's the code I use:

              import {fabric } from "fabric"
              let fabricCanvas = new fabric.Canvas('myCanvas', {
                width: canvasContainer.current.offsetWidth,
                height: canvasContainer.current.offsetHeight,
                isDrawingMode: true
              })
              fabricCanvas.on({
                'touch:gesture': function(e) {
                  console.log(e) // returns empty object wen fired with fabricCanvas.fire("touch:gesture")
                }
              });
              fabricCanvas.fire("touch:gesture") // I can only make the listener fire, by doing this

How can I make the gestures provided work normally?

**Comment by asturur** [comment]
did you try to build from the command line or did you use the custom build from the website?
